tests/ContainsManyTest.php

```diff
@@ -19,8 +19,9 @@
 /**
  * Invoice model.
  *
- * @property Line2  $lines       @Atk\RefMany()
- * @property string $total_gross @Atk\Field()
+ * @property Line2  $lines               @Atk\RefOne()
```

Comment:
always use `refOne`, `refMany` should be used once `load()` returns another instance (otherwise you will not be able to laod a model, you must get it thru iterator first)